README.md

```diff
@@ -79,7 +133,8 @@ Please refer to [CONTRIBUTING.md][contributing].
 [add other sample code repo links]: # (Developer advocacy is open-sourcing a repo that has excellent sample code. The link will be added here.)
 
 [developer guide]: https://docs.snowflake.com/en/developer-guide/snowpark/python/index.html
-[api references]: https://docs.snowflake.com/en/developer-guide/snowpark/reference/python/index.html
+[Snowpark Python api references]: https://docs.snowflake.com/en/developer-guide/snowpark/reference/python/index.html
+[Snowpark pandas api references]: https://docs.snowflake.com/LIMITEDACCESS/snowpark-pandas#api-reference
```

Comment:
Note that we should put up a small PR to fix this link and remove `LIMITEDACCESS` once the docs go live